.github/DISCUSSION_TEMPLATE/help.yml

```
@@ -1,25 +1,36 @@
----
 body:
-  - type: textarea
+  - type: markdown
+    attributes:
+      value: |
+Thanks for taking the time to start a new discussion!
+        
+Before you post:
+- Check if someone has already asked/answered your question in a previous discussion.
+        
+When you're ready to post:
+- Add tags to your discussion (e.g. 'React', 'Vue', 'Vite') to make it clearer for other users.
```

```suggestion
- Add labels to your discussion (e.g. 'React', 'Vue', 'Vite') to make it clearer for other users.
```
A nit to align with GitHub terminology